doc/source/_static/scipy.css

```diff
@@ -0,0 +1,106 @@
+	unicode-range: U+0028, U+0029;
+}
```

Comment:
Can you add a comment for this font. The long URL is hard to verify, will be useful to know where that came from and what it does.